templates/reloader.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: reloader
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: reloader
+  annotations:
+    appversion.kubeaddons.mesosphere.io/dex: "v0.0.49"
+    values.chart.helm.kubeaddons.mesosphere.io/reloader: https://raw.githubusercontent.com/stakater/Reloader/ded923b12a17a0f5c9f37f1c4594331d8d22fa70/deployments/kubernetes/chart/reloader/values.yaml
+    # catalog.kubeaddons.mesosphere.io/addon-revision: "0.0.49-1"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  chartReference:
+    chart: reloader
+    repo: https://stakater.github.io/stakater-charts
```

Comment:
Just curious if there's any particular context about this upstream, do you feel that there's any reason we may potentially may want to fork and bring this in house?

Reply:
So far we're using upstream chart without need to modify it.